incubator/astro/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.controller.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "astro.name" . }}
```

Comment:
RE: RBAC issues - should be `{{ include "astro.fullname" }}`

Reply:
wow, nice find - thank you.  I was testing it with the name astro.  Fixed.